docs/setup-cluster/k8s/resource-caps.rst

```diff
@@ -6,7 +6,8 @@
 
 .. note::
 
-   Managing resource quotas in Determined requires the Determined Enterprise Edition.
+   Auto creating namespaces and managing resource quotas in Determined require the Determined
+   Enterprise Edition.
 
 Determined makes it easier to manage resource quotas by handling them directly, so you don't need to
 manage them separately in Kubernetes. You can manage workspace namespace bindings and resource
```

Comment:
I think we should note that users can only manage resource quotas for auto-created namespaces.

Maybe `Determined makes it easier to manage resource quotas for auto-created namespaces by handling them directly, so you don't need to manage them separately in Kubernetes. You can set workspace-namespace bindings and resource quotas using either the WebUI or CLI.`

Reply:
Since this is mentioned later on in the WebUI seciton, maybe we don't need to say it at the beginning? Not sure if it's worth specifying that detail here